tsconfig.json

```diff
@@ -19,17 +19,11 @@
         "noLib": false,
         "preserveConstEnums": true,
         "sourceMap": true,
-        "watch": false,
-        "typeRoots" : ["node_modules/@types"]
+        "watch": false
     },
-    "include": [
-        "**/*.ts",
-        "node_modules/@types/**/*.d.ts"
-    ],
     "exclude": [
-        "node_modules",
         "example",
-        "test/programs",
+        "test",
```

Comment:
Does this mean we cannot jump to definitions in src from tests or find references in tests from src anymore?